### DISCUSSION TITLE
State of a Job during downtime (Database or JobRunr)

### DISCUSSION THREAD

**Discussion by rejirnair** [jobrunr/jobrunr#716] [General]
We are evaluating JobRunr (latest version - 6.1.0) against MySQL database. I have a question regarding the expected behavior of JobRunr for the following scenario:

A job has started but not completed. The database goes down and consequently JobRunr is killed and eventually restarted when the database comes up. Does the job re-execute from the beginning ? or does the job complete processing during the downtime and the job state is updated after the database comes up.

**Comment by rdehuyss** [comment]
No, processing will start again if it can not update the state in the database.
The job will re-execute from the beginning.
